lib/std/zig/system/x86.zig

```diff
@@ -0,0 +1,664 @@
+const std = @import("std");
+const Target = std.Target;
+const CrossTarget = std.zig.CrossTarget;
+
+fn setFeature(cpu: *Target.Cpu, feature: Target.x86.Feature, enabled: bool) void {
+
+    const idx = @as(Target.Cpu.Feature.Set.Index, @enumToInt(feature));
+
+    if(enabled) cpu.features.addFeature(idx)
+    else cpu.features.removeFeature(idx);
+}
+
+fn hasFeature(cpu: *Target.Cpu, feature: Target.x86.Feature) bool {
+    const idx = @as(Target.Cpu.Feature.Set.Index, @enumToInt(feature));
+    return cpu.features.isEnabled(idx);
+}
```

Comment:
I believe callsites to this can be replaced with `Target.x86.featureSetHas(cpu.features, feature);`